_docs/Maskinporten/maskinporten_feilsoking.md

```diff
@@ -44,33 +45,35 @@ Det som har fungert for andre kunder:
 
 ### Invalid grant
```

Comment:
Ang connection/azure feilen så er den pga porter som var stengt i plattformen av TiE, skal vera opna oppatte no. Forslag til endra tekst: "Vi vet at enkelte kunder som kjører tjenester i Azure noen ganger sliter med å koble seg til maskinporten.no. Dette skal nå være løst.
En typisk feilmelding kan se slik ut
"A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond."
Det som har fungert for andre kunder....